From 3ebabce309334e5d1b8469bc60afed528ae55fc2 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Tue, 1 Jul 2025 12:59:21 +0200
Subject: [PATCH] dt-bindings: media: st: csi: add 'power-domains' property

STM32 CSI may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 CSI.

Fix dtbs_check warning:
 Unevaluated properties are not allowed ('power-domains' was unexpected)

Change-Id: Ia7302873dca03ace6a4d84221721e4e4df3c772d
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/471686
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
---
 Documentation/devicetree/bindings/media/st,stm32-csi.yaml | 3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/devicetree/bindings/media/st,stm32-csi.yaml
+++ b/Documentation/devicetree/bindings/media/st,stm32-csi.yaml
@@ -33,6 +33,9 @@ properties:
   resets:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
   ports:
     $ref: /schemas/graph.yaml#/properties/ports
 
